projects/addon-editor/abstract/editor-adapter.abstract.ts

```diff
@@ -4,6 +4,8 @@ import type {EditorState} from 'prosemirror-state';
 import {Observable, Subject} from 'rxjs';
 
 @Directive()
+// TODO: Class name `TuiEditor` must have one of the following prefixes: Abstract, TuiAbstract
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Good point! We should prepare it before v3.0

Lets mark it as deprecated.
Then create new selector
```ts
export abstract class AbstractTuiEditor extends TuiEditor {}
```

\+ write migration (or add task to `https://github.com/Tinkoff/taiga-ui/issues/1764`)